app/controllers/workouts_controller.rb

```diff
@@ -1,15 +1,13 @@
 class WorkoutsController < ApplicationController
   def new
     # add real template from workouts
-    @template_workout = Workout.first
+    @template_workout = Workout.find_by(status: 'template')
```

Comment:
To be checked by Jesse, I think you're the one who had changed this in your code today